docs/book/src/tasks/experimental-features/cluster-classes.md

```diff
@@ -1,210 +1,21 @@
-# ClusterClass
+# Experimental Feature: ClusterClass (alpha)
 
 The ClusterClass feature introduces a new way to create clusters which reduces boilerplate and enables flexible and powerful customization of clusters.
 ClusterClass is a powerful abstraction implemented on top of existing interfaces and offering a set of tools and operations to streamline cluster lifecycle management while maintaining the same underlying API.
 
-This tutorial covers using Cluster API and ClusterClass to create a Kubernetes cluster and how to perform a one-touch Kubernetes version upgrade.
+**Feature gate name**: `ClusterTopology`
 
-<aside class="note warning">
-
-<h1>Note</h1>
-
-This guide only covers creating a ClusterClass cluster with the CAPD provider.
-</aside>
-
-## Installation
```

Comment:
IMO we can drop this only by augmenting the main quick start with

- a short sentence/paragraph with an example about enabling feature flags just before https://cluster-api.sigs.k8s.io/user/quick-start.html#initialization-for-common-providers
-  note in https://cluster-api.sigs.k8s.io/user/quick-start.html#generating-the-cluster-configuration about using the CAPD template with ClusterClass

Reply:
Agree. I did something like this in this PR: https://github.com/kubernetes-sigs/cluster-api/pull/5839